WTWSMS/decisions/minor_decisions.txt

```diff
@@ -630,7 +637,7 @@ decisions = {
 				factor = 0.001             # Only a small minority of Christians ever go on a pilgrimage
 			}
 			modifier = {
-				factor = 3
+				factor = 50
```

Comment:
It is more frequent _only_ for zealous characters, and _only_ if they take the Theology Focus. That, coupled with the fact that ninety five percent of them won't take the Decision, still makes it extrmely rare.